### PR TITLE
Fix duplicate administrator navbar entry

### PR DESCRIPTION
**Pull request by thomas-burko**
see #5435

### CODE REVIEW

**Comment by gmarziou** [comment]
I'm away from home and cannot test right now but this fix makes sense.

File: generators/client/templates/angular/src/main/webapp/app/shared/auth/_has-any-authority.directive.ts [hunk 1]
```diff
@@ -33,6 +33,7 @@ export class HasAnyAuthorityDirective {
     private updateView(): void {
         this.principal.hasAnyAuthority(this.authorities).then(result => {
             if (result) {
+                this.viewContainerRef.clear();
```
Comment (deepu105) [inline generators/client/templates/angular/src/main/webapp/app/shared/auth/_has-any-authority.directive.ts]: @thomas-burko could you move this line outside the if and remove the else part

**Comment by thomas-burko** [comment]
moved statement outside and removed else part

**Comment by tezcane** [comment]
If I open two different browsers and login one as admin and one as user.  Then the "Administration" menu item shows up on the user view and also disappears from the admin view.

So this does not work:
```
this.viewContainerRef.createEmbeddedView(this.templateRef);
this.viewContainerRef.clear();
```

However, the original seems to cover all the cases I tested:
```
if (result) {
    this.viewContainerRef.clear();
    this.viewContainerRef.createEmbeddedView(this.templateRef);
} else {
    this.viewContainerRef.clear();
}
```

**Pull request merged**